src/index.js

```diff
@@ -25,6 +25,10 @@ import { initCSSPolyfill } from "./scroll-timeline-css"
 
 initCSSPolyfill();
 
+if ([...document.styleSheets].filter(s => s.href !== null).length) {
+  console.warn("Non-Inline StyleSheets detected: ScrollTimeline polyfill currently only supports inline styles within style tags");
```

Comment:
Nit: 80-column character limit.
console warn("...." +
                       "..."); 

to avoid a long line.

Reply:
That's fair! Lemme change that quick 👍